roles/letsencrypt/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+
+maintainer_email: "laskarisn@grnet.gr"
```

Comment:
i'd suggest to not provide a default mail address and have ansible complain if an email address is not provided through group vars

Reply:
Yes, that's correct.
So, it will be provided @ runtime as an ansible-playbook command parameter.

Reply:
It's not necessary to specify it at runtime. I'd suggest to document the var in `defaults/main.yml` in comments:

```
# Email address for important account notifications. For example, Let’s Encrypt will automatically send
# expiry notices to this email address when your certificate is coming up for renewal
#letsencrypt_email: "bob.builder@example.org"
```

and then in `<INV_NAME>/group_vars/letsencrypt/main.yml`:
```
letsencrypt_email: "john.doe@example.com"
```

Reply:
ok, changed it accordingly!